lucene/facet/src/test/org/apache/lucene/facet/FacetTestCase.java

```diff
@@ -56,6 +60,28 @@ public Facets getTaxonomyFacetCounts(TaxonomyReader taxoReader, FacetsConfig con
     return facets;
   }
 
+  public List<List<FacetLabel>> getTaxonomyFacetLabels(TaxonomyReader taxoReader, FacetsConfig config, FacetsCollector fc) throws IOException {
```

Comment:
Thank you for adding this utility method so tests can easily use the new utility class!

Can we rename this to `getAllTaxonomyFacetLabels`, and add javadoc explaining that the outer list is one entry per matched hit, and the inner list is one entry per `FacetLabel` belonging to that hit?

Reply:
done in the next revision.